gallery/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "13.3.0",
+  "version": "13.4.0",
```

IMO this is a patch release not a minor. 
```suggestion
  "version": "13.3.1",
```
(and change the other file too)